uchiwa/helpers.go

```diff
@@ -39,6 +41,15 @@ func findModel(id string, dc string, checks []interface{}) map[string]interface{
 	return nil
 }
 
+// GetIP returns the real user IP address
+func GetIP(r *http.Request) string {
+	if xForwaredFor := r.Header.Get("X-FORWARDED-FOR"); len(xForwaredFor) > 0 {
```

Comment:
Good catch, I'll correct the typo. Concerning the `X-FORWARDED-FOR` header, if it isn't found (`len(xForwardedFor) >  0`), then we get the `r.RemoteAddr` IP address on line 49.